File: pocs/javascript/fixtures/basic-flow.js
function doubleValue(value) {
  return value * 2;
}

const numbers = [2, 4, 6];
let total = 0;

for (let index = 0; index < numbers.length; index += 1) {
  const currentValue = numbers[index];
  const doubledValue = doubleValue(currentValue);

  numbers[index] = doubledValue;
  total += doubledValue;
}

if (total > 20) {
  console.log("Total is greater than 20.");
} else {
  console.log("Total is 20 or less.");
}

console.log(JSON.stringify({ numbers, total }));

File: pocs/python/fixtures/basic-flow.py
import json


def double_value(value):
    return value * 2


numbers = [2, 4, 6]
total = 0

for index in range(len(numbers)):
    current_value = numbers[index]
    doubled_value = double_value(current_value)

    numbers[index] = doubled_value
    total += doubled_value

if total > 20:
    print("Total is greater than 20.")
else:
    print("Total is 20 or less.")

print(
    json.dumps(
        {
            "numbers": numbers,
            "total": total,
        },
        separators=(",", ":"),
    )
)